internal/server/http/handler_test.go

```diff
@@ -69,6 +71,66 @@ func TestImposterHandler(t *testing.T) {
 	}
 }
 
+func TestImposterHandler_Variables(t *testing.T) {
+	var headers = make(map[string]string)
+	headers["Content-Type"] = "application/json"
+
+	responseId1 := "test/testdata/imposters_variables/responses/gopher_1_response.json"
+	responseId2 := "test/testdata/imposters_variables/responses/gopher_2_response.json"
+	responseId1Variable1 := "test/testdata/imposters_variables/responses/gopher_1_1_response.json"
+	responseId1Variable2 := "test/testdata/imposters_variables/responses/gopher_1_2_response.json"
+	responseId1WithoutVariable := "test/testdata/imposters_variables/responses/gopher_1_without_variable_response.json"
```

Comment:
Nit; I think this could be refactored in a way so we use the file contents in the test (instead of the file path), so we don't need an empty file for the case where there's no data, but it's not critical/blocking for merging this PR.